packages/server/controllers/guilds.js

```diff
@@ -14,7 +16,15 @@ async function getGuild(guildId) {
   return query;
 }
 
+async function searchGuild(pattern) {
+  return prisma.$queryRaw`
+    SELECT * FROM guilds 
+    WHERE SIMILARITY(name, ${pattern}) > ${MINIMUM_SIMILARITY}
+    ORDER BY SIMILARITY(name, ${pattern}) DESC;`;
+}
+
```

Comment:
Since these guilds will be displayed in tiny cards as items in a list for search results, we should only select what's necessary to keep the API response data size small.

Keep the fetched guild data to:
- avatar icon
- name
- handler
- guild ID